<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1245</width>
    <height>908</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SWEidMT SS2022</string>
  </property>
  <widget class="QLabel" name="label_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>512</width>
     <height>512</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_image3D">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>90</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>512</width>
     <height>512</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>811</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_startValue">
      <property name="text">
       <string>Start Value: -400</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="horizontalSlider_startValue">
      <property name="minimum">
       <number>-1024</number>
      </property>
      <property name="maximum">
       <number>3071</number>
      </property>
      <property name="value">
       <number>-400</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_windowWidth">
      <property name="text">
       <string>Window Width: 800</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSlider" name="horizontalSlider_windowWidth">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>4095</number>
      </property>
      <property name="value">
       <number>800</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_thresholdValue">
      <property name="text">
       <string>Threshold: 300</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QSlider" name="horizontalSlider_thresholdValue">
      <property name="minimum">
       <number>-1024</number>
      </property>
      <property name="maximum">
       <number>3071</number>
      </property>
      <property name="value">
       <number>300</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_image_Xdir">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>90</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>512</width>
     <height>512</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_image_Zdir">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>500</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>512</width>
     <height>512</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="pushButton_loadImage">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Load .raw image</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/filetype-raw.svg</normaloff>../icons/filetype-raw.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_render3D">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Show 3D image</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/badge-3d.svg</normaloff>../icons/badge-3d.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_regionGrowing">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="toolTip">
       <string>Region Growing</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/arrows-move.svg</normaloff>../icons/arrows-move.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_markers">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Register markers</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/pin-map.svg</normaloff>../icons/pin-map.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_updateCrosssections">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Update Crosssections</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../icons/arrow-clockwise.svg</normaloff>../icons/arrow-clockwise.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>500</y>
     <width>201</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Local Coordinates:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_X">
      <property name="text">
       <string>X: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Y">
      <property name="text">
       <string>Y: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Z">
      <property name="text">
       <string>Z: -</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>500</y>
     <width>191</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Real Coordinates: </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_X_real">
      <property name="text">
       <string>X: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Y_real">
      <property name="text">
       <string>Y: -</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_Z_real">
      <property name="text">
       <string>Z: -</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_layerNumber">
      <property name="text">
       <string>Layer: 0</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="horizontalSlider_layerNumber">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximum">
       <number>399</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>600</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Real world position</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>18</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Z: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_WorldX">
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
       <property name="value">
        <number>-15</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_WorldZ">
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
       <property name="value">
        <number>-57</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_WorldY">
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="stepType">
        <enum>QAbstractSpinBox::DefaultStepType</enum>
       </property>
       <property name="value">
        <number>-65</number>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>mm</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Y: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>730</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Local position</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="3">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>vx</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QSpinBox" name="spinBox_LocalZ">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="value">
        <number>129</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QSpinBox" name="spinBox_LocalX">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="value">
        <number>108</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Z: </string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>vx</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>vx</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QSpinBox" name="spinBox_LocalY">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>400</number>
       </property>
       <property name="value">
        <number>194</number>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Y: </string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_autoUpdateCrosssections">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>870</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Auto update crosssections</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
